components/brave_wallet_ui/components/shared/create-account-icon/create-account-icon.tsx

```diff
@@ -26,20 +37,38 @@ export const CreateAccountIcon = (props: Props) => {
     account,
     size,
     marginRight,
-    round
+    round,
+    externalProvider
   } = props
 
   // Memos
   const orb = useAccountOrb(account)
 
+  if (externalProvider) {
+    const rewardsIcon = getRewardsProviderIcon(externalProvider)
+    return (
+      <ExternalAccountBox
+        size={size}
+        marginRight={marginRight}
+        round={round}
+        provider={externalProvider}
+      >
+        <ExternalAccountIcon
+          src={rewardsIcon}
```

Comment:
consider inlining this value since it is only used once

Reply:
Done